frontend/src/components/SortableTask.tsx

```diff
@@ -57,29 +58,15 @@ export const SortableTask: React.FC<{
           {...provided.draggableProps}
           {...provided.dragHandleProps}
         >
-          <div className={css.leftSideDiv}>
-            {task.name}
+          <div className={css.leftSideDiv}>{task.name}</div>
+          <div className={css.rightSideDiv}>
             <div>
-              <TaskRatingsText task={task} />
+              {unversionedTasks &&
+                unversionedTasks.some((obj) => obj.id === task.id) && (
```

Comment:
Tässä toimisi optional chaining oikein hyvin
```suggestion
              {unversionedTasks?.some((obj) => obj.id === task.id) && (
```